### PR TITLE
Sound: removed play() in favor of play_file()

### PR DESCRIPTION
**Pull request by dwalton76**
For issue #532

### CODE REVIEW

File: ev3dev2/sound.py [hunk 1]
```diff
@@ -247,16 +247,27 @@ def play_note(self, note, duration, volume=100, play_type=PLAY_WAIT_FOR_COMPLETE
...
-        :param play_type: The behavior of ``play`` once playback has been initiated
+        :param play_type: The behavior of ``play_file`` once playback has been initiated
         :type play_type: ``Sound.PLAY_WAIT_FOR_COMPLETE``, ``Sound.PLAY_NO_WAIT_FOR_COMPLETE`` or ``Sound.PLAY_LOOP``
 
         :returns: When ``Sound.PLAY_NO_WAIT_FOR_COMPLETE`` is specified, returns the spawn subprocess from ``subprocess.Popen``; ``None`` otherwise
         """
+        if not 0 < volume <= 100:
+            raise ValueError('invalid volume (%s)' % volume)
+
+        if wav_file.endswith(".mp3"):
```
Comment (WasabiFan) [inline ev3dev2/sound.py]: Why not require that it _does_ end in `.wav` rather than _not ending_ in `.mp3`?

**Review by WasabiFan — COMMENTED** [review]
I think there are code examples in this file and maybe in other docs pages which use the old method name; those also need to be updated. Looks good to me otherwise.

File: ev3dev2/sound.py (hunk 1 above)
Comment (dwalton76) [reply ev3dev2/sound.py]: I was worried about breaking things if someone had a wav file that did not end in .wav. That would be pretty silly though...will change it to enforce .wav.

Comment (WasabiFan) [reply ev3dev2/sound.py]: I come from the Windows world where file extensions are used for a lot of logic, so maybe my POV is different. wav files without extensions might be common, I'm not sure.

**Comment by WasabiFan** [comment]
I think example code needs to be updated too.

**Comment by dwalton76** [comment]
Fixed the example in the docstring

**Pull request merged**